apps/memcached.go

```diff
@@ -0,0 +1,51 @@
+package apps
+
+import (
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator"
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/otel"
+)
+
+type MetricsReceiverMemcached struct {
+	confgenerator.ConfigComponent `yaml:",inline"`
+
+	confgenerator.MetricsReceiverShared `yaml:",inline"`
+
+	Endpoint string `yaml:"endpoint" validate:"omitempty,hostname_port|file"`
```

Comment:
Should we do `hostname_port|startswith=/` instead of `hostname_port|file` here as well?

See a similar conversation in https://github.com/GoogleCloudPlatform/ops-agent/pull/218/files/8620d7199eb6a617dc369657eade36956d49c609

Reply:
good point, updated.